contracts/contracts/strategies/DAIMigrationStrategy.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { InitializableAbstractStrategy } from "../utils/InitializableAbstractStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IDaiUsdsMigrationContract {
+    function daiToUsds(address usr, uint256 wad) external;
+
+    function usdsToDai(address usr, uint256 wad) external;
+}
+
+contract DAIMigrationStrategy is InitializableAbstractStrategy {
+    address public immutable dai;
+    address public immutable usds;
+
+    constructor(
+        BaseStrategyConfig memory _baseConfig,
+        address _dai,
+        address _usds
+    ) InitializableAbstractStrategy(_baseConfig) {
+        dai = _dai;
+        usds = _usds;
+    }
+
+    function initialize(address _governorAddr)
+        external
+        onlyGovernor
+        initializer
+    {
+        _setGovernor(_governorAddr);
```

Comment:
Since `DAIMigrationStrategy` is not proxied, I think setting the governor would be better in the constructor

Reply:
That'd make calling `intialize` complicated (since it'll have to be part of the proposal)

Reply:
Actually, think we might not even need a `initialize`. All of that could go into the `constructor`. Let me change that